reactioncommerce/reaction-next-starterkit
# Add inventory status badges to variants and options

Add inventory status badges to variants/options lists. 

**NOTE**: The badge style will be updated when the designers finish the design of the PDP page.

### Testing
1. Add a new product with a variant and an option and publish the product.
2. By default the variant will have a backorder badge.
3. Verify a backorder badge is visible on the new product variant in the  grid and PDP.
4. Toggle the backorder switch to turn off backordering
5. Grid and PDP should display a "Sold Out" badge
6. Repeat steps 3-5 for the variant option.
